### ISSUE TITLE
Userquery Connection failure

### ISSUE THREAD

**Issue by glaubervila** [linea-it/dri#1292]
In userquery when executing the query below, and trying to check the error "Connection Failure" appears.

Investigating the error happens because the query check is taking longer than the default browser timeout which is 30 seconds.

```
SELECT coadd_object_id, ra, dec, mag_auto_g, mag_auto_r, flags_g, class_star_g
FROM DES_ADMIN.DR1_MAIN
WHERE flags_g < 4
AND mag_auto_g - mag_auto_r > -0.3
AND mag_auto_g - mag_auto_r < 0.0
AND class_star_g > 0.8 
```

**Comment by glaubervila** [comment]
Closed in v2.3

**Comment by glaubervila** [comment]
The error keeps happening, the query is never submitted, apparently the part of the submission that checks if the query is valid is causing a time out.
